src/DDTrace/Processing/TraceAnalyticsProcessor.php

```diff
@@ -13,33 +13,48 @@
  */
 final class TraceAnalyticsProcessor
 {
+    /**
+    * @param bool|float $value
+    */
+    public static function normalizeAnalyticsValue(&$array, $value)
```

Comment:
I think I'd call this `$metrics` and update the DocBlock. :)

```suggestion
    public static function normalizeAnalyticsValue(&$metrics, $value)
```